src/main/java/sonardenemeleri/sonar/App.java

```diff
@@ -7,7 +7,12 @@
 public class App 
 {
     public static void main( String[] args )
-    {
+    {final int deneme;
+	int target = -5;
+	int num = 3;
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "deneme2" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)